### PR TITLE
Delete unused seed file support from PEWriter

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/dlls/mscorpe/pewriter.h [hunk 1]
```diff
@@ -221,73 +207,13 @@ class PEWriterSection : public PESection {
...
-protected:
-
-    PEDecoder *             m_pSeedFileDecoder;
-    IMAGE_SECTION_HEADER *  m_pSeedSectionHeader;
-
 };
 
 inline DWORD PEWriter::getSectionAlignment() {
-    return VAL32(m_ntHeaders->OptionalHeader.FileAlignment);
+    return VAL32(m_ntHeaders->OptionalHeader.SectionAlignment);
```
Comment (jkotas) [inline src/coreclr/dlls/mscorpe/pewriter.h:213]: This is fixing a copy&paste bug. (This method is not actually reachable, so the bug is harmless.)

**Comment by jkotas** [comment]
This dead code caused confusion during a security investigation, so it is better to just delete it.

File: src/coreclr/dlls/mscorpe/iceefilegen.cpp [hunk 2]
```diff
@@ -34,22 +34,13 @@ HRESULT ICeeFileGen::CreateCeeFile (HCEEFILE *ceeFile)
...
-
-HRESULT ICeeFileGen::CreateCeeFileEx2 (HCEEFILE *ceeFile, DWORD createFlags, LPCWSTR seedFileName)
 {
     if (!ceeFile)
         return E_POINTER;
 
     CeeFileGenWriter *gen = NULL;
     HRESULT hr;
-    IfFailRet(CeeFileGenWriter::CreateNewInstanceEx(NULL, gen, createFlags, seedFileName));
+    IfFailRet(CeeFileGenWriter::CreateNewInstance(NULL, gen, createFlags));
```
Comment (elinor-fung) [inline src/coreclr/dlls/mscorpe/iceefilegen.cpp:43]: Is the `CCeeGen*` ever non-NULL? I see `CreateCeeFileFromICeeGen`, but I don't see anything calling that.

**Review by elinor-fung — APPROVED** [review]

File: src/coreclr/dlls/mscorpe/iceefilegen.cpp (hunk 2 above)
Comment (jkotas) [reply src/coreclr/dlls/mscorpe/iceefilegen.cpp]: Deleted

**Pull request merged**